### ISSUE TITLE
Tutorial Translation

### ISSUE THREAD

**Issue by ron4ex** [reactjs/gu.reactjs.org#10]
Tutorial translation is going to be a major undertaking. I think it will need to be done in smaller chunks. Creating a separate issue to track its progress.

Sections:
* [ ] Before We Start the Tutorial ( @3sanket3 )
* [ ] Setup for the Tutorial
* [ ] Overview
* [ ] Completing the Game
* [ ] Adding Time Travel

**Comment by error404-d** [comment]
do you still need a contributor

**Comment by ridays2001** [comment]
Uh, is this still being done? I think I may be able to help. Gujarati is my mother tongue...

**Comment by harshilparmar** [comment]
@ridays2001 Please take any page that you want to work on here #1 👍

**Comment by ridays2001** [comment]
Oh, Okie. Will check that. Thanks 👍🏻
